extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -69,6 +69,7 @@ export const databaseReferencesNodeName = localize('databaseReferencesNodeName',
 export const sqlConnectionStringFriendly = localize('sqlConnectionStringFriendly', "SQL connection string");
 export const yesString = localize('yesString', "Yes");
 export const noString = localize('noString', "No");
+export const noStringDefault = localize('noStringDefault', "No (default)");
```

Comment:
That typically doesn't work for localized strings since different locales can put words in a sentence in different orders. (think RTL languages for example)